libcrux-ml-dsa/src/sample.rs

```diff
@@ -143,17 +143,17 @@ pub(crate) fn rejection_sample_less_than_eta<const ETA: usize>(
 #[allow(non_snake_case)]
 #[inline(always)]
 fn sample_error_ring_element<const ETA: usize>(seed: [u8; 66]) -> PolynomialRingElement {
```

Comment:
This looks very similar to `sample_challenge_ring_element`. Can we re-use code?

Reply:
I'm not sure, the basic structure is indeed similar (squeeze some bytes, pass these bytes to a function, and repeat if need be); but I think thematically they're different, `sample_error_ring_element` starts at 0 and systematically fills the ring element, whereas `sample_challenge_ring_element` is sprinkling a fixed number of positive or negative 1's randomly across the array.